test/route.go

```diff
@@ -59,3 +62,22 @@ func UpdateBlueGreenRoute(t *testing.T, clients *Clients, names, blue, green Res
 	}
 	return clients.ServingClient.Routes.Patch(names.Route, types.JSONPatchType, patchBytes, "")
 }
+
```

Comment:
Since we are editing the file, it seems that the `CreateBlueGreenRoute` is not used by anyone. Should we nuke it?

Reply:
Not in this PR, feel free to open a cleanup PR to adress that please.